File: test/fixtures/r/post_with_data_with_percent_sign.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = "secret=*%5*!"

res <- httr::POST(url = "http://localhost:28139/post", httr::add_headers(.headers=headers), body = data)

File: test/fixtures/ruby-httparty/post_with_data_with_percent_sign.rb
require 'httparty'

url = 'http://localhost:28139/post'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = 'secret=*%5*!'
res = HTTParty.post(url, headers: headers, body: body)
